bindings/python/py_src/safetensors/torch.py

```diff
@@ -84,7 +84,7 @@ def _tobytes(tensor: torch.Tensor) -> bytes:
     import ctypes
     import numpy as np
 
-    length = math.prod(tensor.shape)
+    length = np.prod(tensor.shape).item()
```

Comment:
Ok, I had removed it because PT doesn't technically require numpy (I still had to keep it do get the underlying buffer without copying)